### ISSUE TITLE
Compare results graphically

### ISSUE THREAD

**Issue by Narasimha1997** [kostya/benchmarks#421]
Is there any standard format you are following to publish results?

If yes, we can build a tool (or a web page) that interactively compares different measurements and programming languages.

**Comment by nuald** [comment]
Not really, the raw results are analyzed by the Ruby script that generates Markdown tables inserted into README. However, I can publish the raw results, so they could be analyzed with Jupyter or any other data science system. If they have any standards, I could change the raw results format too as it's just a matter of Ruby script tweaks.

**Comment by mcourteaux** [comment]
The charts are nice, but I have two remarks:
 - Probably all charts need logarithmic scaling for the vertical axis.
 - The x-labels are not complete: more than half of the bars don't have a label.
